icons/LICENSE

```diff
@@ -0,0 +1,395 @@
+﻿Attribution 4.0 International
```

Comment:
I am not a lawyer. Can someone explain what implications this massive wall of text has?

Reply:
I'm not a lawyer as well, but the conditions are explained in detail in section 3, or for a shorter summary see for example https://choosealicense.com/licenses/cc-by-4.0/ .

From that text I'm pretty sure that all conditions are met here:
* gave attribution to original author and included a hyperlink
* stated modifications
* there is no copyright notice provided in the original repository, so no further files need to be included

Actually I believe we don't have to include the whole license file, as it only says to retain
> iii. a notice that refers to this Public License;

but I wanted to be on the very safe side even though a link should be sufficient. If we wanted we could use another license for the modified PNG icons too, but I think this creative commons license is no bad choice.

Also we are not allowed to advertise the modified PNG icons under the name of Microsoft (no trademark use), and we can't blame Microsoft if the icons will crash or destroy our computer somehow (no liability and warranty).

Reply:
Now I actually wonder whether we need a notice and license for the already existing lightbulb.png too, or where exactly did it come from in #489 ?

Reply:
We should probably copy the very bottom section (Legal notice) of the Codicons' Readme too. I'm gonna add it to the attribution and modifications notice.

Reply:
> From that text I'm pretty sure that all conditions are met here:

Yes, I think so too.